android/src/test/java/vn/luongvo/kmm/survey/android/test/CoroutineTestRule.kt

```diff
@@ -0,0 +1,38 @@
+package vn.luongvo.kmm.survey.android.test
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.test.*
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+import vn.luongvo.kmm.survey.android.util.DispatchersProvider
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class CoroutineTestRule : TestWatcher() {
+
+    internal val testDispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()
```

Comment:
`TestCoroutineDispatcher` is deprecated. Please consider using `UnconfinedTestDispatcher` or `StandardTestDispatcher`